runtime/vara/src/lib.rs

```diff
@@ -116,6 +116,10 @@ pub use extensions::DisableValueTransfers;
 
 mod migrations;
 
+#[link_section = "wasm_runtime_version"]
+static _RUNTIME_VERSION: [u8; const_str::to_byte_array!(env!("SUBSTRATE_CLI_IMPL_VERSION")).len()] =
```

Comment:
Could we use `env!().len()` here, or do you find it unsafe for some Unicode cases? Anyway, it will be detected in compile time.
```suggestion
static _RUNTIME_VERSION: [u8; env!("SUBSTRATE_CLI_IMPL_VERSION").len()] =
```

Reply:
I think we can, but it to me it looks more consistent and more accurate. Do not see why we might want to go with `env!(...).len()`

Reply:
It is shorter and more readable. We may use `env!("...").as_bytes().len()` to be more precise.